client/src/components/Metrics/MetricsPersonal/MetricsPersonal.tsx

```diff
@@ -32,18 +34,29 @@ const MetricsPersonal = (): ReactElement => {
   const { isFetching: isFetchingCryptoValues } = useCryptoValues(displayCurrency);
   const { isFetching: isFetchingMetricsPersonalDataRewardsUsage } =
     useMetricsPersonalDataRewardsUsage();
+  const { data: totalPatronDonations, isFetching: isFetchingTotalPatronDonations } =
+    useTotalPatronDonations();
 
-  const isLoading = isFetchingCryptoValues || isFetchingMetricsPersonalDataRewardsUsage;
+  const isLoading =
+    isFetchingCryptoValues ||
+    isFetchingMetricsPersonalDataRewardsUsage ||
+    isFetchingTotalPatronDonations;
+
+  const wasUserEverAPatron = totalPatronDonations && totalPatronDonations.numberOfEpochs > 0;
 
   return (
     <div className={styles.root} id={METRICS_PERSONAL_ID}>
       <div className={styles.divider} />
       <MetricsHeader title={t('yourMetrics')} />
       {isConnected ? (
         <MetricsGrid className={styles.grid} dataTest="MetricsPersonal__MetricsGrid">
-          <MetricsPersonalGridAllocations isLoading={isLoading} />
+          <MetricsPersonalGridAllocations
+            isLoading={isLoading}
+            size={wasUserEverAPatron ? 'XL' : 'L'}
```

Comment:
`XL` value should be replaced with `custom` and an additional `classname` -> like TOP X Projects tile. There is only one tile with such dimensions.

Reply:
What's the difference here? Instead of custom define it as XL, is that a problem?

Reply:
I don't understand, can you please rephrase?

Reply:
Done.